### ISSUE TITLE
Sub-tasks

### ISSUE THREAD

**Issue by upstreamplugin** [upstreamplugin/UpStream#47]
One user requested the ability to have sub-tasks underneath main tasks

**Comment by tardis221** [comment]
@upstreamplugin, I look forward to seeing this extension added. It will be extremely useful.

**Comment by upstreamplugin** [comment]
Thanks @tardis221 

The plan here is to build out a system of custom fields. If all goes well, we'll use this as the basis for creating parent / child tasks.

**Comment by upstreamplugin** [comment]
Closing in favor of dealing with this via Custom Fields

**Comment by upstreamplugin** [comment]
I guess the only issue here with the Custom Fields approach is that you can still move the Task to "Complete", even if the correct fields have been filled in.

**Comment by upstreamplugin** [comment]
From a customer ...

" I have to deal with non standard subtasks, i.e. for the project "New formats" I need
. Format Selection
a) mockup editing
b) functionality
c) ...
. Supplier Selection
a) Supplier A
b) Supplier B
c) ....."

**Comment by stevejburge** [comment]
From a customer

> I want to set a task or a milestone as a "Must be completed before to preceed with the next task / milestone" ?
> 
> In the task list I would see what task I can do because all the previous task are completed and what are waiting for previous.
